### PR TITLE
Review Map control.

### PR DESCRIPTION
**Pull request by mariaterzieva12**

### CODE REVIEW

File: controls/map/layers/overview.md [hunk 1]
```diff
@@ -1,26 +1,27 @@
...
-description: Check our &quot;Layers Overview&quot; documentation article for Telerik Map for .NET MAUI control.
+description: Learn more about the Map Layers used in Telerik UI for .NET MAUI Map control.
 position: 0
 slug: map-layers-overview
 ---
 
 # .NET MAUI Map Layers
 
-The Map control visualizes spatial data using *layered architecture*. It could display spatial data points (e.g. location of cities), polylines (e.g. road connections), or polygons (shape of countries or continents) in a *geographic coordinate system*. In this coordinate system, a point on the surface of the earth is defined using latitudes and longitudes.
+The Map control visualizes spatial data using *layered architecture*. It can display spatial data points
```
Comment (yordan-mitev) [inline controls/map/layers/overview.md:11]: - Don't use italics for emphasis
- Don't use a line break before ending the sentence.

File: controls/map/layers/overview.md [hunk 2]
```diff
@@ -1,26 +1,27 @@
...
+(location of cities), polylines (road connections), or polygons (shape of countries or continents) in a *geographic coordinate system*. In this coordinate system, a point on the surface of the earth is defined using latitudes and longitudes.
 
 * **Latitude**: This is measured as the angle from the equatorial plane to the line drawn from the center of the earth to the point on the surface. Usually latitude values range from 90° (toward the North Pole and suffixed with "N") to -90° (towards the South Pole and suffixed with "S").
```
Comment (yordan-mitev) [inline controls/map/layers/overview.md:14]: - Don't use bold for the terms.
- Use an em dash as a separator for the definition lists.

File: controls/map/overview.md [hunk 3]
```diff
@@ -1,34 +1,34 @@
...
-description: Check our &quot;Overview&quot; documentation article for Telerik Map for .NET MAUI.
+description: Try now the Telerik UI for .NET MAUI Map control delivering a visualization for rich spatial data.
 position: 0
 slug: map-overview
 ---
 
 # .NET MAUI Map Overview
 
-**Telerik Map for .NET MAUI** is a data visualization control whose primary purpose is to visualize rich spatial data. The control provides visualization of ESRI shapefiles that consist of geometric objects, such as lines, polylines and polygons. Such objects are commonly used to display various schemes, for example floor plans and seats distribution, all the way to parts of maps for countries, roads, rivers, etc.
+Telerik Map for .NET MAUI is a data visualization control whose primary purpose is to visualize rich spatial data. The control provides visualization of ESRI shapefiles that consist of geometric objects, such as lines, polylines and polygons. Such objects are commonly used to display various schemes, for example floor plans and seats distribution, all the way to parts of maps for countries, roads, rivers, etc.
```
Comment (yordan-mitev) [inline controls/map/overview.md:11]:
```suggestion
The Telerik UI for .NET MAUI Map is a data visualization control whose primary purpose is to visualize rich spatial data. The control provides visualization of ESRI shapefiles that consist of geometric objects, such as lines, polylines and polygons. Such objects are commonly used to display various schemes, for example floor plans and seats distribution, all the way to parts of maps for countries, roads, rivers, etc.
```

File: controls/map/overview.md [hunk 4]
```diff
@@ -1,34 +1,34 @@
...
 
-![Map Overview](images/map_overview.png)
+![.NET MAUI Map Overview](images/map_overview.png)
 
 ## Key Features
 
-* [Shapefile visualization]({% slug map-layers-overview %})&mdash;The Map can display rich spatial data from ESRI shapefiles. Each shapefile should be loaded and configured through a ShapefileLayer instance added to the Layers collection of the control.
+* [Shapefile visualization]({% slug map-layers-overview %})&mdash;The Map can display rich spatial data from ESRI shapefiles. Each shapefile have to be loaded and configured through a `ShapefileLayer` instance added to the Layers collection of the control.
```
Comment (yordan-mitev) [inline controls/map/overview.md:17]:
```suggestion
* [Shapefile visualization]({% slug map-layers-overview %})&mdash;The Map can display rich spatial data from ESRI shapefiles. Each shapefile has to be loaded and configured through a `ShapefileLayer` instance added to the Layers collection of the control.
```

File: controls/map/pan-and-zoom.md [hunk 5]
```diff
@@ -23,32 +23,32 @@ Here is a quick snippet how `InteractionMode` is applied:
 
 <snippet id='map-interaction-mode-xaml' />
 
-where the Source of the MapShapeReader should be defined as well:
+where the Source of the `MapShapeReader` have to be defined as well:
```
Comment (yordan-mitev) [inline controls/map/pan-and-zoom.md:26]: Use complete sentences.

File: controls/map/selection.md [hunk 6]
```diff
@@ -36,13 +36,13 @@ Add the namespace:
...
 ```
 
-Check below how RadMap with a few selected shapes will look like:
+Check below how `RadMap` with a few selected shapes will look like:
 
-![Map Multiple Selection](images/map_multiple_selection.png)
+![.NET MAUI Map Multiple Selection](images/map_multiple_selection.png)
 
-## Example with programatically shape selection
+## Example with programmatically shape selection
```
Comment (yordan-mitev) [inline controls/map/selection.md:43]:
```suggestion
## Example with Programmatical Shape Selection
```

File: controls/map/styling/shapesstyles.md [hunk 7]
```diff
@@ -18,33 +18,33 @@ The Map `ShapefileLayer` exposes `ShapeStyle` property which helps you to style
...
+The snippet below shows how `ShapeStyle` property can be applied:
 
 <snippet id='map-styling-shapeslstyle' />
 
 Here is the result:
 
 ![Map Shapes Styling](../images/map_styling_shapestyle.png)
 
-where the `Source` and the `DataSource` of the MapShapeReader should be set to a .shp and .dbf files, respectively:
+where the `Source` and the `DataSource` of the `MapShapeReader` have to be set to a `.shp` and `.dbf` files, respectively:
```
Comment (yordan-mitev) [inline controls/map/styling/shapesstyles.md:29]: Use complete sentences.

File: controls/map/commands.md [hunk 8]
```diff
@@ -1,39 +1,39 @@
...
 
 * `ZoomOutCommand`
 
-You can manually call these commands, for example on button click action, to zoom-in or zoom-out respectively, the shapes displayed in RadMap.
+You can manually call these commands, for example on button click action, to zoom-in or zoom-out respectively, the shapes displayed in `RadMap`.
 
 Following is a quick example on how the commands of the Map control can be called from external UI:
 
-1. The RadMap definition:
+1. The `RadMap` definition:
```
Comment (didiyordanova) [inline controls/map/commands.md:21]: bold list

**1.**
**2.**, etc

File: controls/map/styling/shapesstyles.md [hunk 9]
```diff
@@ -18,33 +18,33 @@ The Map `ShapefileLayer` exposes `ShapeStyle` property which helps you to style
...
 <snippet id='map-interactionmode-settintsource' />
 
 ## Style Selector for Shapes
 
-Using the `ShapeStyleSelector` property of the `ShapefileLayer` you could implement conditional styling. 
+Using the `ShapeStyleSelector` property of the `ShapefileLayer` you can implement conditional styling. 
 
 The example below shows how to apply different styles to shapes according to certain property value of each shape.
 
 1. First, create the selector class which inherit from `MapShapeStyleSelector`:
```
Comment (didiyordanova) [inline controls/map/styling/shapesstyles.md:39]: use bold list here: 
**1.**
**2.**, etc

and remove the space before the snippet id

**Pull request merged**